### PR TITLE
Manage category

### PR DESCRIPTION
**Pull request by vovanquang12cntt**

### CODE REVIEW

**Comment by vovanquang12cntt** [comment]
ready

File: app/views/admin/categories/_category.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= increase_stt(params[:page], Settings.per_page, @stt) %></td>
+  <td><%= category.address %></td>
+  <td><%= link_to t("admin.tour.edit"), edit_admin_category_path(category) %> |
+    <%= link_to t("admin.tour.delete"), admin_category_path(category), method: :delete, data:{confirm: t("admin.tour.confirm_dele")} %></td>
```
Comment (tranducquoc) [inline app/views/admin/categories/_category.html.erb]: dài quá

**Comment by tranducquoc** [comment]
commented

**Comment by vovanquang12cntt** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**